helm/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: sql-exporter
+description: Database agnostic SQL exporter for Prometheus
+type: application
+version: 0.1.0
+appVersion: 0.12.3
+keywords:
```

Comment:
@allanger Would you mind adding yourself to the maintainers list? I'm referring to the reference field below:

```
maintainers: # (optional)
  - name: The maintainers name (required for each maintainer)
    email: The maintainers email (optional for each maintainer)
    url: A URL for the maintainer (optional for each maintainer)
```

Reply:
I've added myself there